modules/selenoid/src/test/java/it/selenoid/SelenoidClipboardTest.java

```diff
@@ -47,4 +48,14 @@ public void setAndGetClipboardMultilineContent() {
     clipboard().shouldHave(content(multilineText));
     assertThat(clipboard().getText()).isEqualTo(multilineText);
   }
+
+  @Test
+  public void setAndGetClipboardMultilineContentWithBecause() {
+    String multilineText = "John\nWick\r\nThe\nGreat\r";
+    clipboard().setText(multilineText);
+    assertThatThrownBy(() ->
+                         clipboard().shouldHave(content(multilineText + "test").because("Content don`t right")))
+      .hasMessageContaining("");
```

Comment:
I don't get what you want to test here. An empty set of char values exists in any string.

Reply:
Forgot)